login-workflow/src/components/SetPassword/SetPassword.tsx

```diff
@@ -84,14 +88,24 @@ export const SetPassword: React.FC<React.PropsWithChildren<SetPasswordProps>> =
                 inputRef={passwordRef}
                 label={newPasswordLabel}
                 value={passwordInput}
-                onChange={(evt: ChangeEvent<HTMLInputElement>): void => onPassChange(evt.target.value)}
+                error={shouldValidatePassword && !isValidPassword()}
+                sx={TextFieldStyles(theme)}
+                {...passwordTextFieldProps}
+                onChange={(evt: ChangeEvent<HTMLInputElement>): void => {
+                    // eslint-disable-next-line no-unused-expressions
+                    passwordTextFieldProps?.onChange && passwordTextFieldProps.onChange(evt);
```

Comment:
This line is not helpful as we do nothing when this expression is true or false. Can you please elaborate more on why we are using this?

Reply:
We are checking for the onChange event passed or not in passwordTextFieldProps, if it's false we are not going to call that callBack function.